packages/realm-server/tests/realm-server-test.ts

```diff
@@ -168,6 +168,14 @@ module('Realm Server', function (hooks) {
       testRealmURL.href,
       'realm url header is correct',
     );
+    assert.true(
+      response.get('X-realm-public-readable') != undefined,
+      'realm is public readable',
+    );
+    assert.true(
+      response.get('X-realm-public-writable') != undefined,
+      'realm is public writable',
+    );
```

Comment:
Would these and below work with `assert.ok(response.get('X-realm-public-writable'))` instead? It seems a bit more concise to just check for truthiness.